test_scripts/API/Registration/011_Registered_app_with_appID_has_NULL_permissions_in_Policy_Table.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- Check that SDL does not the application registration with appID which has "null" permissions in Policy Table.
+-- In case:
+-- 1) Application registered with appID which has "null" permissions in Policy Table.
+-- SDL does:
+-- 1) Successfully register a mobile application
```

Comment:
@GetmanetsIrina Description is controversial. Do SDL allow (or disallow) registration of app with `null` permissions?